FPWD/2023-05-23/index.html

```diff
@@ -252,6 +252,33 @@
         "name": "W3C"
       }
     },
+    {
+      "id": "https://w3c-ccg.github.io/data-integrity-spec/",
```

Comment:
this is outdated, but can be fixed later... see the references to it in https://github.com/w3c/vc-di-bbs/blob/main/index.html#L100

Reply:
updating now